node/packages/parser/src/ethereum/yearn.ts

```diff
@@ -4,9 +4,19 @@ import shapeShiftRouter from './abi/shapeShiftRouter'
 import yearnVault from './abi/yearnVault'
 import { ethers } from 'ethers'
 import { SHAPE_SHIFT_ROUTER_CONTRACT } from './constants'
-import { getSigHash, getYearnTokenVaultAddresses } from './utils'
+import { getSigHash } from './utils'
+import { Yearn, ChainId } from '@yfi/sdk'
```

Comment:
Someday we can get import order linting, but just more "only in my head ocd patterns" for you, I typically import in the following order:
`@scopedPackages`
`thirdPartyPackages`
`../../relativePathFurthest`
...
`./relativePathClosest`

Feel free to update or feel free to tell me to chill :joy:

Reply:
I can get on board with that 😉

Fixed up in https://github.com/shapeshift/unchained/pull/284/commits/0ff68c2eb17e35722614eb688ae805f8d989976f